packages/cdc-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,82 @@
+@use "../../scss/global.scss";
+@use "uswds-core" as *;
+
+.checkbox-container {
+  display: flex;
```

Comment:
When there is no description, the label looks a bit off-center
<img width="85" alt="image" src="https://github.com/CDCgov/cdc-react/assets/98106992/d7abf61d-af30-445d-bfa0-cbf71b5551bd">